Sources/Kitura/Kitura.swift

```diff
@@ -105,6 +105,7 @@ public class Kitura {
     // MARK: Start Servers
     
     /// Start the Kitura framework.
+    /// By default, the Kitura framework process will exit with a code of zero, even if one or more of the servers fails to start. To enable the Kitura framework process to exit with a non-zero exit code set the `withStatus` parameter to true.
```

Comment:
By default, the Kitura framework does not call `exit` at all - the reason a Kitura process typically exits with 0, is that the `run()` function completes (as no servers remain to listen on), and `Kitura.run()` is typically the final line of code for the main thread to execute.  Once the last line completes, the default action is that the process ends with a return code of zero.

If the user is able to determine that there was an error, then they could call `exit` themselves with a non-zero code, ie:
```swift
let numberOfFailures = Kitura.runWithStatus()
exit(numberOfFailures)
```